.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,97 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ "main" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
```

Comment:
Just a question, but could we get rid of the “if pull request” checks if we dropped this?

Reply:
Probably. Will check that in 1 hour